### DISCUSSION TITLE
我这里有完整的中国文化大革命文库

### DISCUSSION THREAD

**Discussion by RobertBornsWoodward** [banned-historical-archives/banned-historical-archives.github.io#2766] [Q&A] [answered]
请问如何上传？

**Comment by banned-historical-archives** [comment, marked as answer]
你可以先上传到你的仓库

**Reply by RobertBornsWoodward** [reply]
https://mega.nz/file/KdZ0kTSA#WwMt-HX_PuDkHxK_nzwFAEiUTsjqe47xKME-cmuBdlo

**Comment by ghost** [comment]
你可以先上传到你的仓库

**Comment by PirateBook1** [comment]
@RobertBornsWoodward 如果 github 不会使用可以往一些境外网盘传

**Reply by RobertBornsWoodward** [reply]
https://mega.nz/file/KdZ0kTSA#WwMt-HX_PuDkHxK_nzwFAEiUTsjqe47xKME-cmuBdlo

**Reply by banned-historical-archives** [reply]
> https://mega.nz/file/KdZ0kTSA#WwMt-HX_PuDkHxK_nzwFAEiUTsjqe47xKME-cmuBdlo

好像是第一版的内容，这个版本我们已经录入了

**Reply by PirateBook1** [reply]
@banned-historical-archives @RobertBornsWoodward 不好确定是不是第一版，有些文章附有出处，但是无论是不是这个 docx 一股脑堆在一起的排版很难解析

**Reply by ghost** [reply]
> https://mega.nz/file/KdZ0kTSA#WwMt-HX_PuDkHxK_nzwFAEiUTsjqe47xKME-cmuBdlo

@PirateBook1 
be6b756ece37f6c40fa01c56dc980537（Prolet）
be6b756ece37f6c40fa01c56dc980537（MEGA）
http://www.metools.info/other/o21.html （云计算MD5检测工具。）
经过MD5、stat.st_size的对比，它就是Prolet上的中国文化大革命文库第一版，不是好像，它就是，算是发了一份已经完全解析过的文件了。你可以和Prolet上的 https://github.com/ProletRevDicta/Prolet/raw/master/A%20%E6%97%A0%E4%BA%A7%E9%98%B6%E7%BA%A7%E6%96%87%E5%8C%96%E5%A4%A7%E9%9D%A9%E5%91%BD/%E4%B8%AD%E5%9B%BD%E6%96%87%E5%8C%96%E5%A4%A7%E9%9D%A9%E5%91%BD%E6%95%B0%E6%8D%AE%E5%BA%93%20%E5%AE%8B%E6%B0%B8%E6%AF%85.docx 对比一下MD5。
